tutor/src/screens/task/ux.js

```diff
@@ -195,10 +193,19 @@ export default class TaskUX {
           `[data-task-step-id="${this.currentStep.id}"]`
         );
       }
+    } else {
+      this._stepIndex = index;
+      CenterControls.currentTaskStep = this.currentStep;
     }
   }
 
+  @computed get isApiPending() {
+    return get(this.currentStep, 'api.isPending', false);
+  }
+
   @computed get canGoForward() {
+    if (this.isApiPending) { return false; }
```

Comment:
I thought we wanted them to be able to move forward while the answer is pending?

Reply:
we did but if the step type changes then it's possible the code would start to render one type and then it'd update afterwards.  I suspect that might be part of the `map on undefined` issue